src/tss2-esys/api/Esys_NV_UndefineSpaceSpecial.c

```diff
@@ -212,6 +212,9 @@ TSS2_RC
 Esys_NV_UndefineSpaceSpecial_Finish(
     ESYS_CONTEXT *esysContext)
 {
+     ESYS_TR nvIndex;
```

Comment:
this looks like 5 spaces to me, should be 4?

Reply:
done